src/Reflection/ParametersAcceptorSelector.php

```diff
@@ -234,7 +234,7 @@ public static function selectFromArgs(
 				];
 			}
 
-			if (isset($args[0]) && (bool) $args[0]->getAttribute(ArrayFindArgVisitor::ATTRIBUTE_NAME)) {
```

Comment:
we already work on line 82 with `$arrayMapArgs = $args[0]->value->getAttribute(ArrayMapArgVisitor::ATTRIBUTE_NAME);` without a outer `isset`, meaning the following `isset` checks are useless